src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown

```diff
@@ -167,7 +167,6 @@ navigator.bluetooth.requestDevice({ filters: [{ services: ['battery_service'] }]
   return server.getPrimaryService('battery_service');
 })
 .then(service => {
-  if (!service) throw 'Battery Service not found';
```

Comment:
Nope, because if `service` is `null`, it will raise an exception that will be caught by the last `catch`.